theme/templates/email/signup_verify.txt

```diff
@@ -0,0 +1,8 @@
+{% load i18n %}
+{% block main %}
+{% trans "Welcome to HydroShare. This email address was used to request an account on HydroShare." %}
+{% trans "If you originated the request, please use the link below to verify your email address and activate your account." %}
+http://{{ request.get_host }}{{ verify_url }}
```

Comment:
The security implication is minor, but exists. It still allows the verification link to be sent unencrypted before the redirect happens, so an attacker in the middle could intercept that, not that its very likely because what real value would that have? Still, if its possible to avoid, all the better.